internal/command/command.go

```diff
@@ -424,14 +431,40 @@ func UsageMetrics(command *cobra.Command, wg *sync.WaitGroup) {
 
 // GlobalFlags contains all global flags definitions.
 type GlobalFlags struct {
-	Filter           string
-	Format           string
-	Save             string
-	Host             string
-	HostNetworkKey   string
-	Log              string
-	Network          string
-	Yes              bool
-	ConfigPaths      []string
-	SkipVersionCheck bool
+	Filter                     string
+	Format                     string
+	Save                       string
+	Host                       string
+	HostNetworkKey             string
+	Log                        string
+	Network                    string
+	Yes                        bool
+	ConfigPaths                []string
+	SkipVersionCheck           bool
+	SkipContractMigrationCheck bool
+}
+
+func checkContractMigrations(state *flowkit.State, logger output.Logger, flow flowkit.Services) {
+	contractStatuses, err := validator.NewValidator(github.NewClient(nil).Repositories, flow.Network(), state, logger).GetContractStatuses()
+	if err != nil {
+		logger.Error(fmt.Sprintf("failed to validate contracts: %s", err))
+		return
+	}
+
+	for _, contract := range contractStatuses {
+		if contract.IsFailure() {
+			logger.Error(fmt.Sprintf(
+				"Contract %s has failed the last emulated migration\n"+
+					" - Account: %s\n"+
+					" - Contract: %s\n"+
+					" - Error: %s\n",
+				contract.ContractName,
```

Comment:
Also, this is a lot to show. Is it better just to say like "3 contracts with errors found since last migration," and then a way to see more?

Reply:
ya i agree i think it's quite a bit. Maybe we can list the contracts and then just like to the migration data?

Reply:
updated now

![image](https://github.com/onflow/flow-cli/assets/6364887/7d381f7c-8612-4067-87af-652ff3b6eda5)


I/ll update the original PR screenshot as well

Reply:
What does it look like if there are multiple that fail?

Reply:
![image](https://github.com/onflow/flow-cli/assets/6364887/e727b925-bfc7-4927-aff4-4f13195942de)

i'll update the initial PR  to show that as well if anyones curious